Competition2017/src/competition/subsystems/collector/CollectorSubsystem.java

```diff
@@ -16,10 +16,10 @@
 public class CollectorSubsystem extends BaseSubsystem {
     private static Logger log = Logger.getLogger(CollectorSubsystem.class);
     
-    private final DoubleProperty intakePowerProperty;
-    private final DoubleProperty ejectPowerProperty;
+    public final DoubleProperty intakePowerProperty;
```

Comment:
I am reluctant to use public properties. I think we have the infrastructure to change values without access to the property objects, and we could also subclass these things for tests and have the subclasses expose either the properties or setters publicly.